library/std/src/os/fd/owned.rs

```diff
@@ -296,6 +297,7 @@ impl From<fs::File> for OwnedFd {
 
 #[stable(feature = "io_safety", since = "1.63.0")]
 impl From<OwnedFd> for fs::File {
+    /// Creates a [`File`](fs::File) instance from an existing file descriptor.
```

Comment:
I feel like this should also say "Takes ownership ..." like the other comments added here, so that we make it clear this implementation moves that ownership from the OwnedFd rather than e.g. dup + close or something.

(Same in a few other, similar cases below)